File: backend/src/main/java/com/example/backend/dto/UserEditDto.java
package com.example.backend.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserEditDto {
    String firstName;
    String lastName;
    String address;
    String photo;
}

File: app/src/main/java/com/example/dropletbarterapp/mainscreens/profile/dto/UserEditDto.kt
package com.example.dropletbarterapp.mainscreens.profile.dto

import com.google.gson.annotations.SerializedName
import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor

class UserEditDto(
    @SerializedName("firstName")
    var firstName: String? = null,
    @SerializedName("lastName")
    var lastName: String? = null,
    @SerializedName("address")
    var address: String? = null,
    @SerializedName("photo")
    var photo: ByteArray? = null
)